sympy/utilities/tests/test_lambdify.py

```diff
@@ -829,3 +831,13 @@ def test_lambdify_dummy_arg():
     d2 = Dummy('x')
     f2 = lambdify(d2, d2 + 1)
     assert f2(2) == 3
+
+def test_issue_14283():
+    prntr = PythonCodePrinter()
+    expr1 = x/0
+    func = lambdify(x, expr1, "numpy")
+    assert prntr.doprint(func(1)) == "float('nan')"
+
+    expr2 = oo*-1
```

Comment:
And `-oo` here.

Reply:
The changes have been implemented.
@asmeurer . Thank you.